src/dex-ui/services/MirrorNodeService/MirrorNodeService.ts

```diff
@@ -24,6 +24,18 @@ import {
 import govenorAbi from "../abi/GovernorCountingSimpleInternal.json";
 import { ProposalType } from "../../store/governanceSlice";
 
+const govenorAbiSingatureMap = new Map<string, any>();
+const web3 = new Web3();
+
+function loadGovernorAbiEventSignatures() {
+  govenorAbi.abi.forEach((eventAbi: any) => {
```

Comment:
Can we create a type in the types.ts file for the ABI objects? It's ok if the values are all conditional for now:

For example,
```
interface ABI {
      anonymous?: boolean;
      inputs?: Array<any>;
      name?: string;
      type?: string;
      outputs?: Array<any>
      etc...
}
```